core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -229,8 +229,7 @@ import scala.concurrent.duration._
 
     case ResetPersistenceId(_, tp @ TagProgress(pid, _, _)) =>
       log.debug("Resetting persistence id {}. TagProgress {}", pid, tp)
-      become(
-        writeInProgress(buffer.filterNot(_._1.persistenceId == pid), tagPidSequenceNrs + (pid -> tp.pidTagSequenceNr)))
+      become(writeInProgress(buffer.filterNot(_._1.persistenceId == pid), tagPidSequenceNrs + (pid -> tp.pidTagSequenceNr), awaitingFlush))
```

Comment:
This is a bug that doesn't happen as we don't send ResetPersistenceId while doing a full flush (done in migration) but just in case we change that fixed it here too